superset-frontend/package.json

```diff
@@ -48,11 +48,16 @@
     "cover": "cross-env NODE_ENV=test NODE_OPTIONS=\"--max-old-space-size=4096\" jest --coverage",
     "dev": "webpack --mode=development --color --watch",
     "dev-server": "cross-env NODE_ENV=development BABEL_ENV=development node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=development",
-    "eslint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx --quiet",
```

Comment:
To avoid breaking CI pipelines for downstream Superset users who may still be calling npm run eslint, can we keep the eslint script for now and have it call npm run lint under the hood? We can then add a note to UPDATING.md that the command is deprecated and will be removed in a future release. This gives consumers a safer migration path while we switch to OXC.

Reply:
Added some cleanup to make sure other scripts and docs no longer refer to the `eslint` npm script.